client/src/common/components/DropdownMenu.jsx

```diff
@@ -0,0 +1,91 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+
+import MenuIcon from 'assets/images/ellipsis-v-solid.svg';
+
+class DropdownMenu extends Component {
+  state = {
+    hideMenu: true
+  };
+
+  hideMenu = () => {
+    this.setState({ hideMenu: true });
+    document.removeEventListener('click', this.hideMenu);
+    document.removeEventListener('keydown', this.onPressEscape);
+  };
+
+  showMenu = () => {
+    this.setState({ hideMenu: false });
+    document.addEventListener('click', this.hideMenu);
+    document.addEventListener('keydown', this.onPressEscape);
+  };
+
+  onPressEscape = e => {
+    if (e.keyCode === 27) this.hideMenu();
+  };
```

Comment:
this can be written in one line `onPressEscape = e => e.keyCode === 27 ? this.hideMenu() : null`